File: backend/enoughhome/portable_git.go
package enoughhome

import (
	"os"
	"path/filepath"
)

// PortableGitDir returns the path where PortableGit is provisioned on Windows.
func PortableGitDir() string {
	if la := os.Getenv("LOCALAPPDATA"); la != "" {
		return filepath.Join(la, "enough", "git")
	}
	return filepath.Join(HomeDir(), "git") // fallback
}

File: backend/hollowhome/portable_git.ts
import path from "node:path";
import { home_dir } from "./home";

/** Portable Git install path on Windows. */
export const portable_git_dir = (): string => {
  const la = process.env.LOCALAPPDATA;
  if (la !== undefined && la !== "") {
    return path.join(la, "hollow", "git");
  }

  return path.join(home_dir(), "git");
};
